README.md

```diff
@@ -0,0 +1,118 @@
+# Clerk User Import Script
+
+## Description
+
+This repository contains a script that takes a JSON file as input, containing a list of users, and creates a user in Clerk using Clerk's backend API. The script respects rate limits and handles errors.
+
+## Getting Started
+
+Clone the repository and install the dependencies.
+
+```bash
+git clone github.com/clerk/migration-script
+
+cd migration-script
+
+bun install
```

Comment:
Let's not, I rather it be more Node-oriented, Bun is a dependency so people don't need to install new tooling just whatever Node package manager they have. We can also gitignore the lockfile (normal practice in OSS and certain tooling)

Reply:
You want to ignore all lock files or just the bun one?

Reply:
ignore all of the `*-lockfile` or whatever